lib/puppet/feature/manages_local_users_and_groups.rb

```diff
@@ -0,0 +1,6 @@
+require 'puppet/util/feature'
+require 'puppet/util/libuser'
+
+Puppet.features.add(:manages_local_users_and_groups) {
+  Puppet.features.libuser? || Facter.value(:kernel) == "AIX"
```

Comment:
Is this needed as the feature looks to be enabled only on aix and libuser cases already?

Reply:
It seems it is not needed anymore.